samples/maps/demo/basic-map/demo.details

```diff
@@ -0,0 +1,12 @@
+---
+name: World High Resolution
```

Comment:
While it _is_ implied, I would maybe sneak in a "Map" here; perhaps "High Resolution Word Map"?

Reply:
Actually I don't think it is worthwhile mentioning High Resolution. That doesn't really have anything to do with what we are trying to accomplish with this demo. What about "Basic world map"?